### PR TITLE
Draft: Add macOS paths to importDWG.py

### PR DESCRIPTION
**Pull request by Roy-043**
This PR adds adds handling of macOS paths to importDWG.py.

Paths taken from this forum topic:
https://forum.freecad.org/viewtopic.php?t=79125&start=10

- [x]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

---

### CODE REVIEW

**Comment by yorikvanhavre** [comment]
LGTM!

File: src/Mod/Draft/importDWG.py [hunk 1]
```diff
@@ -199,26 +198,29 @@ def get_oda_converter():
...
         if os.path.exists(odadir):
             for sub in os.listdir(odadir):
                 path = odadir + "/" + sub + "/" + "ODAFileConverter.exe"
                 if os.path.exists(path) and os.path.isfile(path):
                     return path
+    elif platform.system() == "Linux":
+        path = "/usr/bin/ODAFileConverter"
+        if os.path.exists(path) and os.path.isfile(path):
+            return path
+    else: # for macOS
```
Comment (Nailig) [inline src/Mod/Draft/importDWG.py:212]: There should be a check if macOS, else unknown system

Comment (Nailig) [reply src/Mod/Draft/importDWG.py:212]: Ok. Let’s say we disagree about programming practices.

Comment (yorikvanhavre) [reply src/Mod/Draft/importDWG.py:212]: In any case I don't know what we would do right now on a system that is not mac, linux or windows... I think if someone comes up with a new config, then we see about what paths to use...

Comment (luzpaz) [reply src/Mod/Draft/importDWG.py:212]: @Roy-043 FreeBSD fixes are submitted to FreeCAD from time to time. So while it is unofficial, it still supported  
https://github.com/search?q=repo%3AFreeCAD%2FFreeCAD%20freebsd&type=code

**Pull request merged**